src/core_atmosphere/physics/mpas_atmphys_control.F

```diff
@@ -473,7 +476,22 @@ subroutine physics_compatibility_check(dminfo, blockList, streamManager, ierr)
 
      call mpas_dmpar_max_real(dminfo, maxvar2d_local, maxvar2d_global)
 
-     if (maxvar2d_global <= 0.0_RKIND) then
+     !
+     ! The GWDO check below can fail on regional simulations that are completely above
+     ! water. So, check to see if the simulation is completely above water and do not
+     ! throw the error if it is.
+     !
+     call mpas_pool_get_array(sfc_inputPool, 'iswater', iswater_lu)
+     call mpas_pool_get_array(sfc_inputPool, 'ivgtyp', ivgtyp)
+     if (all(ivgtyp(:) == iswater_lu)) then
```

Comment:
I think we need to restrict the range of cells that we're checking to `(1:nCells)`, or even better, to `(1:nCellsSolve)`, to avoid mis-interpreting the garbage cell as a non-water point.

@MiCurry Just in case you happen to see this, would you mind if I pushed an amended commit to this PR branch?

Reply:
@mgduda feel free! I've added you as a contributor on my fork, so all my branches are belong to you!

Reply:
Many thanks!